backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -476,28 +479,78 @@ public UUID getPermissibleOrgId(UUID orgId) {
     return orgId != null ? orgId : getCurrentOrganization().getInternalId();
   }
 
-  @AuthorizationConfiguration.RequireGlobalAdminUser
-  public List<UUID> getOrgAdminUserIds(UUID orgId) {
+  private List<ApiUser> getOrgAdminUsers(UUID orgId) {
     Organization org =
         organizationRepository.findById(orgId).orElseThrow(NonexistentOrgException::new);
```

Comment:
I know this was in the extant code (which high key I might have written since my name is next to the git blame 😅), but I think there's an opportunity for improvement here to return an empty list on the event that there isn't an org rather than throwing, especially since we're expanding the scope of where this method gets called to outside just the GQL mutation context. 

Since we're returning a list of users, the state of "no org admins found"--whether it be because there is an org and no admins or because there just isn't an org--is represented just as well semantically by an empty list as it is an error. However, an empty list is way simpler for the caller to handle rather than a bespoke error since in most cases I'd think we'd just be iterating over the list as needed anyways. Lmk what you think

Reply:
OOO good call -- I'll fix this up!